### ISSUE TITLE
only reading postgres.conf and not what is in the live database

### ISSUE THREAD

**Issue by ikzelf** [timescale/timescaledb-tune#80]
Hi, first of all, thanks, it is a nice tool.
I happen to have a configuration where conf.d is used  and timescales-tune seems not to recognise this. A nice improvement could be to read the values from the live database and maybe write the changes to conf.d/13_tuned.conf.

I feel this would make life a bit easier. 13 is obviously a random choice and should be the highest number so it overrules all previous settings.

**Comment by fastman** [comment]
yeah, I agree. I'd be nice if tune was reading config files from `include` and `include_dir`.
